src/main/java/io/opensergo/OpenSergoConfigKindRegistry.java

```diff
@@ -42,6 +43,7 @@ public final class OpenSergoConfigKindRegistry {
         registerConfigKind(ConfigKind.THROTTLING_STRATEGY, ThrottlingStrategy.class);
         registerConfigKind(ConfigKind.CONCURRENCY_LIMIT_STRATEGY, ConcurrencyLimitStrategy.class);
         registerConfigKind(ConfigKind.CIRCUIT_BREAKER_STRATEGY, CircuitBreakerStrategy.class);
+        registerConfigKind(ConfigKind.TRAFFIC_ROUTER_STRATEGY, RouteConfiguration.class);
```

Comment:
`RouteConfiguration.class` should refers to `opensergo`,  but the import Class is `io.envoyproxy.envoy.config.route.v3.RouteConfiguration`

Reply:
We didn't change envoy's RouteConfiguration at all, so we just reuse the RouteConfiguration provided by envoy